.changeset/large-geese-play.md

```diff
@@ -0,0 +1,16 @@
+---
+"astro": minor
+---
+
+Adds a new `inferRemoteSize` function that can be used to infer the dimensions of a remote image outside of the Image and Picture components. This is useful for when you need to know the dimensions of an image for styling purposes or to calculate different densities for responsive images.
```

Comment:
```suggestion
Adds a new `inferRemoteSize()` function that can be used to infer the dimensions of a remote image.

 Previously, the ability to infer these values was only available by adding the [`inferSize`] attribute to the `<Image>` and `<Picture>` components. Now, you can access this data outside of these components. 
 
 This is useful for when you need to know the dimensions of an image for styling purposes or to calculate different densities for responsive images.
```

What do you think about something like this? I think maybe it highlights for people skimming and thinking, wait, I thought I could already infer remote dimensions?" and emphasizes that this is a function, and "not just in the components anymore" aspect.

Reply:
Fine with me! One note is that inferSize can also be used with getImage, not just the components